base/multi.jl

```diff
@@ -871,27 +1020,61 @@ function local_remotecall_thunk(f, args, kwargs)
     return ()->f(args...; kwargs...)
 end
 
+"""
+    remotecall(f, w::LocalProcess, args...; kwargs...) -> Future
```

Comment:
This form of the API is for internal use. `Worker` is not exported and `LocalProcess` should be unexported.